crates/nu-path/src/helpers.rs

```diff
@@ -10,6 +10,42 @@ pub fn config_dir() -> Option<PathBuf> {
     dirs_next::config_dir()
 }
 
+pub fn vendor_completions_dirs() -> Vec<PathBuf> {
+    let vendor_completions_fn = |mut path: PathBuf| {
+        path.push("nushell");
+        path.push("vendor_completions.d");
```

Comment:
Not a fan of ending directories with and extension.

Reply:
This is an established convention. See https://lists.debian.org/debian-devel/2010/04/msg00352.html. I got the inspiration for this from fish, which uses the same name for it's completions, but we can also use something else if there's consensus that this would be a bad name.

Reply:
Thanks for the education. However, nushell is not a unix-only shell. It's not an established convention on Windows. IMO, at a minimum, it needs to be changed for Windows.

Reply:
Is there a similar established convention for Windows? Because if there isn't, I'd be inclined to just keep it this way.

Reply:
For me, I'm not interested in trying to change Windows to become more line *nix. Most Windows users will see it and not understand it. The only Windows convention is powershell, which we shouldn't interfere with.

Others, feel free to join in.

Reply:
system directories ending in `.d` have always felt weird and old to me, i would just use `nushell/completions/`

note that i'm speaking for Linux, not for Windows, as @fdncred said, it might feel even weirder for Windows folks :confused:

Reply:
okay, I changed it to `nushell/completions/`